src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
+		if (symbol.length >= 11) {
```

Comment:
can we use a ternary operator here? 

Something like 

``` symbol.length >= 11 ?  do x : do y; ```

Reply:
@Seroxdesign not too sure this works.

<img width="723" alt="Screenshot 2022-12-25 at 14 34 09" src="https://user-images.githubusercontent.com/48827267/209468081-c3a53966-5de5-49f5-9681-e1c2e314406f.png">

Reply:
nvm fixed